### PR TITLE
Fix GH#19306: String data "Always open" field cut off

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #19306 
at least after a revert to factory settings, else it might still use the previous too narrow dialog width
or after sizing the dialog manually

### CODE REVIEW

File: src/notation/view/widgets/editstringdata.ui
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstringdata.ui:27]: This string is not used but set programatically, so translating it isn't needed

File: src/notation/view/widgets/editstringdata.ui
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstringdata.ui:32]: same

**Review by Eism — APPROVED** [review]

**Comment by shoogle** [comment]
@Jojo-Schmitz, a tooltip is welcome in a separate PR, after this is merged. See https://github.com/musescore/MuseScore/issues/19362#issuecomment-1713850372.

**Comment by abariska** [comment]
@Jojo-Schmitz Could you add minimum size values to this dialog to make both columns are visible and their names are fully readable by default, so user won't be able to collapse it more than that min state?
Like this:
abariska 2023-09-11 at 18 02 53

**Comment by Jojo-Schmitz** [comment]
Not sure what a reasonalbe min value might be, as it dependes on the translated strings

**Comment by abariska** [comment]
Because after this fix it could be like this. 
abariska 2023-09-12 at 00 50 58
To me it looks like horizontal scroll is useless (if not to say worse). So I propose to add a minimum size for the dialog or remove horizontal scroll

File: src/notation/view/widgets/editstringdata.ui
Comment (shoogle) [inline src/notation/view/widgets/editstringdata.ui:33]: Missing `<b>Always open</b><br>` at the beginning.

You can use curly quotes here too: `“0” ‘always open’`. But wait for a reply to https://github.com/musescore/MuseScore/issues/19362#issuecomment-1715401132.

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstringdata.ui:33]: Curly quotes around numbers lead to compiler warnings:
"decimal digit terminates octal escape sequence (C4125)"

Comment (shoogle) [reply src/notation/view/widgets/editstringdata.ui:33]: Strange. I see people complaining about that online, but I don't see it myself with MSVC.

Did you enable extra warnings? (How?)

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstringdata.ui:33]: No extra warnings enabled. Just the same build I use since long. VS 2022

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstringdata.ui:33]: If I do it via C++ code (i.e. in editstringdata.cpp, to have it on every string line) I get
The file contains a character starting at offset 0xb14 that is illegal in the current source character set (codepage 65001). (C4828)

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstringdata.ui]: Ah, seems I've overcome that now.

Comment (Jojo-Schmitz) [reply src/notation/view/widgets/editstringdata.ui]: I guess adding the `qtrc()` did the trick (of marking the strings being Unicode?)

**Comment by Jojo-Schmitz** [comment]
Looks like this now:
image

File: src/notation/view/widgets/editstringdata.cpp
Comment (shoogle) [inline src/notation/view/widgets/editstringdata.cpp:53]: Let's use single quotes everywhere (`‘0’`) and shorten the ending to just "such as the theorbo".

Comment (shoogle) [reply src/notation/view/widgets/editstringdata.cpp:53]: You can also store this string in a `QString` variable and then reference it below so you only have to write it once.

File: src/notation/view/widgets/editstringdata.cpp
Comment (shoogle) [inline src/notation/view/widgets/editstringdata.cpp:53]: Missing "the" before theorbo.

**Comment by abariska** [comment]
@Jojo-Schmitz now it's definitely better, but, please, get rid of this blank area. The Pitch column should end accurately before scroll bar
abariska 2023-09-13 at 14 50 14

**Comment by Jojo-Schmitz** [comment]
done, I believe

**Review by abariska — APPROVED** [review]

**Review by Eism — APPROVED** [review]

**Pull request merged**
